### PR TITLE
samples: mesh: nrf52: to solve pending issues after merging PR:9997 

### PR DESCRIPTION
**Pull request by vikrant8052**
https://github.com/zephyrproject-rtos/zephyr/pull/9997

If user aggresively vary the state then previous mechanism
failed to publish status of other bounded states since it
has to publish multiple things back to back.
This PR repair the things.

Signed-off-by: Vikrant More <vikrant8051@gmail.com>

### CODE REVIEW

**Comment by vikrant8051** [comment]
Kindly do not merge as of now.
Waiting for PTS outcome to improve further.

**Comment by vikrant8051** [comment]
Kindly do not merge as of now.
Waiting for PTS outcome to improve further.

**Comment by jhedberg** [comment]
> Kindly do not merge as of now.
> Waiting for PTS outcome to improve further.

If you don't flag your PR as "RFC" or "DNM" I will merge them if I'm happy and Shippable doesn't complain.

Please set up PTS for yourself (get your employer to become a SIG member if they're not - the minimum required membership is free) and test your code with it instead of relying on someone else to do it.

**Review by jhedberg — APPROVED** [review]

**Comment by vikrant8051** [comment]
@jhedberg 
That's sound good.
---------------------------------------------------------------------------------------
Please note some **cons** regarding **onoff_level_lighting_vnd_app**:

1) Currently this app support only one transaction at a time.

2) It is not able to Publish unsolicited message when user set Publish
    address for Server Model. (For this only I've created this PR)

**Pull request merged**
